### DISCUSSION TITLE
GitHub packages version view does not show tags

### DISCUSSION THREAD

**Discussion by ppaulweber** [community/community#135772] [Packages] [answered]
### Select Topic Area

Bug

### Body

Dear GitHub community,

was there recently a change to the GitHub package container registry view?
Because I noticed that the tags are not rendered in the web UI of tagged images anymore and just depicts the digest image SHA only.   

![image](https://github.com/user-attachments/assets/feeb7375-1e0a-4797-8840-38ece870f975)

**Comment by RyStanley89** [comment, marked as answer]
Hello @ppaulweber Just to let you know that this was a bug which is now fixed. Please check again and let us know if the tags still aren't present.

**Comment by ppaulweber** [comment]
Thx @RyStanley89 for the quick answer. I can confirm that all the packages from private and public repositories in question show the image tags again :smiley:
